addons/last-edit-tooltip/addon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Last Edit Tooltip",
+  "description": "Hover over a project's share date for information on when it was last edited.",
+  "credits": [
+    {
+      "name": "TheColaber",
+      "link": "https://www.scratch.mit.edu/users/TheColaber"
```

Comment:
Remove `www`? The link appears to be broken with that subsubdomain

Reply:
www makes all scratch links redirect to the homepage for some reason.